src/DecodeStream/DecodeStream.cpp

```diff
@@ -206,23 +207,21 @@ try { //Giant try block around all code to get error messages
       // TODO: Add proper handling
       break;
     }
-
-    DataUnit du;
-    inStream >> du;
+    
+    DataUnit d;
```

Comment:
The source originally used `du` for this variable; to reduce the quantity of unrelated changes you may prefer to use that instead?